crates/ecs/src/lib.rs

```diff
@@ -1455,7 +1453,7 @@ mod tests {
         let vecs_u32 = world
             .borrow_component_vecs_with_signature::<u32>(&[TypeId::of::<u32>()])
             .unwrap();
-
+        eprintln!("vecs_u32 = {:#?}", vecs_u32);
```

Comment:
just a small tip, you can inline printable variables as long as they're not a more complex expression:
```suggestion
        eprintln!("vecs_u32 = {vecs_u32:#?}");
```